dev/tasks/build.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint node: true */
+
+'use strict';
+
+module.exports = function( grunt ) {
+	grunt.registerTask( 'build', 'Build a release out of the current development code.', function() {
+		var done = this.async();
+		module.exports.build( done );
+	} );
+};
+
+// Exports the build method so it can be used from plain node code as well.
+module.exports.build = function( done ) {
```

Comment:
Can't we create a class for the builder? And then wouldn't it be good to have unit tests for builder too?

It's not super complicated right now, but there's no doubt that it will grow, and tdd would be very handy.